extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -139,6 +140,10 @@ describe('amp-a4a', () => {
       return ampdocService.getAmpDoc(element);
     };
     element.isBuilt = () => {return true;};
+    element.getLayoutBox = () => {
+      const visible = element.style.display != 'none';
+      return layoutRectLtwh(0, 0, visible ? 200 : 0, visible ? 50 : 0);
```

Comment:
This entangles checking `display:none` with dimension checks.  Can we have separate tests for the different cases?  (Just paranoia: I want to be sure that chain is blocked regardless of whether web designer sets styling or dimensions.)